docs/.rstcheck.cfg

```diff
@@ -1,7 +1,7 @@
 [rstcheck]
 ignore_directives=automodule,http:get,tabs,tab,prompt,http:patch,http:post,http:put,http:delete
 ignore_roles=djangosetting,setting,featureflags,buildpyversions
-ignore_substitutions=org_brand,com_brand,:smile:
+ignore_substitutions=org_brand,com_brand,:smile:,:arrows_counterclockwise:,:heavy_plus_sign:,:tada:,:heart:,:pencil2:
```

Comment:
Would be nice if we could just say ignore `:*:`

Reply:
That won't work apparently https://github.com/myint/rstcheck/blob/3f92957478422df87bd730abde66f089cc1ee19b/rstcheck.py#L265-L266 and rstcheck seems to be unmaintained.